src/scales/scale.time.js

```diff
@@ -16,42 +16,42 @@ var INTERVALS = {
 	millisecond: {
 		common: true,
 		size: 1,
-		steps: factorize(1000)
```

Comment:
`factorize` is unused after these changes?

Reply:
It's used in `core.scale.js` instead now